### DISCUSSION TITLE
Programming

### DISCUSSION THREAD

**Discussion by Salis09** [community/community#65150] [Programming Help]
What is java

**Comment by cesarcoello** [comment]
Java is a high-level, object-oriented programming language. It was developed by Sun Microsystems, now owned by Oracle. A prominent feature is its ability to achieve portability, which means that a program written in Java can work on different platforms without the need for major modifications.

**Comment by mateuseap** [comment]
In summary, Java is a programming language that functions as a shared medium for programmers to communicate with computers. It aids in the development of various computer applications and enables their execution on diverse computer types without significant modifications. Much like the common language in an office, Java assists programmers and computers in collaborating effectively!

I've aimed to provide a straightforward answer, avoiding intricate technical details. If you're interested in delving deeper into Java, you might find the links below to be useful:

- https://aws.amazon.com/pt/what-is/java/: An easily understandable article crafted by the AWS team, covering Java's fundamentals;
- https://www.ibm.com/topics/java: Another user-friendly resource from IBM, shedding light on the basics of Java;
- https://www.geeksforgeeks.org/introduction-to-java/: An informative post containing a touch of technical explanation.

**Comment by r00ott** [comment]
You mean java or js ?
